docs/content/components/text-field.mdx

```diff
@@ -0,0 +1,98 @@
+---
+title: TextField
+figma: https://www.figma.com/file/DFKyTGHAoDxOsUBPszLLxP/%F0%9F%8F%B5%EF%B8%8FMarigold?node-id=340%3A0
+---
+
+# TextField
+
+## Description
+
+The `TextField` components allows user so enter text with a keyboard. It consists of a label, an input field and a help text. The later also includes error messages.
+
+```tsx
+<TextField label="Try me" description="You should really try this!" />
+```
+
+```tsx onlyCode
+import { TextField } from '@marigold/components';
+```
+
```

Comment:
Figma Link between import and props

Reply:
until we have a good place for it I'll use the frontmatter. I think we have the usage planned in the respective issue